src/infra/src/table/short_urls.rs

```diff
@@ -242,12 +226,8 @@ pub async fn get_expired(
     if let Some(limit) = limit {
         res = res.limit(limit as u64);
     }
-    let res = res.into_model::<ShortId>().all(client).await;
-
-    match res {
-        Ok(records) => Ok(records.iter().map(|r| r.short_id.clone()).collect()),
-        Err(e) => Err(Error::DbError(DbError::SeaORMError(e.to_string()))),
```

Comment:
This logic moved to [this trait implementation](https://github.com/openobserve/openobserve/pull/5100/files#r1847373471) so it should now happen automatically when `?` converts the error.